### ISSUE TITLE
Fader/volume units in LMMS effects inconsistency- %, dB, dBFS, and dBv

### ISSUE THREAD

**Issue by Monospace-V** [LMMS/lmms#6316]
### Bug Summary
Inconsistency in volume representation

1. #### dB vs %
When "display volume as dbFS" setting is disabled, volume should show in percentage values,
the LMMS effects (like Equalizer, Compressor, Crossover equaliser) faders still show volume in decibels.
While I can understand that things like Threshold (for compression) need to be in dB, and the equalizer's individual bands limit you to a  -12 dB maximum meaning that 0% as the baseline value can be misleading, the _main_ input/output gain faders go from +30 dB to -60 dB, which allows % to be used with reasonable straightforwardness (0% to 100% to 150% range, I believe, would be only slightly inaccurate).
![image](https://user-images.githubusercontent.com/76674645/154804977-3c74648e-94cd-4a67-b9e6-406143a25723.png)
![image](https://user-images.githubusercontent.com/76674645/154804996-0786dce9-2dd0-4edf-88c7-c6079714115c.png)

I understand that the output gain knob at least for compressors is typically used for makeup gain and by extension preferable in the same volume unit as the threshold.

Also, multitap echo and spectrum analyzer, where volume representation is in dB. 
![image](https://user-images.githubusercontent.com/76674645/154805129-280b1227-c0d8-48e4-8a50-410b1254eb83.png)
![image](https://user-images.githubusercontent.com/76674645/154805231-57e4a08f-3f7b-4962-817e-4504d85a1bf6.png)
At least for the former, a percent representation of volume would make more sense for users with Display as dbFS disabled.

2. #### volume units inconsistency within decibels
It is possible I haven't listed all variations, because I use % values display, and dbFS display enabled may show me more, but:
The gain on the equalizer's individual bands is in dB (decibels)
The overall input/output gain on the equalizer is in dBv (decibel volts? Not entirely sure)
The compressor and spectrum analyser's default units are, however, in dbFS (decibels relative to full scale)

(Also, on the multitap echo effect,
![image](https://user-images.githubusercontent.com/76674645/154805660-04756142-042f-44cb-89c5-5c302af72c30.png)
The + on the 0 annoys me to no ends because 0 is neither positive nor negative. It's an unnecessary character. This is just a slightly annoying minor thing, but it'd be very nice if it were fixed.)

**Comment by dj-pixus** [comment]
Maybe not fully related to this problem, but I think dBFS should not be used as a unit in the first place. The only context it would be accepted is when setting an oscillator's amplitude. Any other volume slider/knob just adjusts a signal that already exists, therefore the correct unit would be dB.
Just a small side note.

**Comment by Monospace-V** [comment]
Addition to this issue:
in dbFS mode,
![image](https://github.com/user-attachments/assets/eac9e245-02db-4afd-8fbd-397c7b6a7a6f)
the slider in the mixer is dbFS output
But the "Set Value" dialogue box is percentage
![image](https://github.com/user-attachments/assets/b8a72dbf-be4b-4c6c-9f33-c8b39c68f441)

This happens with the mixer, but not with oscillator volume.
![image](https://github.com/user-attachments/assets/4e7f0331-9126-42e1-827f-cf6793e2fc92)
Is the version I am using.

**Comment by michaelgregorius** [comment]
I agree that the units are currently inconsistent. It would also be better if the faders worked on a dB scale as well. They are currently linear, i.e. the middle position is 100% (0 dB), the top position is 200% (6 dB) and the bottom position is 0% (-inf dB). The meters are already rendered in dB though.

Letting the users enter values in dB units in the dialog brings the challenge of how they should enter "-inf" dB. Currently they can enter 0 which silences the channel. Letting users enter something like "-inf" means that the dialog might have to be extended to also accept textual input. Another question is if that representation should be translated. For example "-inf" is currently translated to "-unendl." in German. Should users have to enter "-unendl." to silence a channel with the German translation? Or "-inf"?

**Comment by michaelgregorius** [comment]
Pull request https://github.com/LMMS/lmms/pull/7636 changes the faders to use a dB scale like all other DAWs and addresses some of the concerns mentioned in this issue. Please refer to the PR for more information.

**Comment by bratpeki** [comment]
Covering this after #7636 is merged!

**Comment by michaelgregorius** [comment]
> Covering this after [#7636](https://github.com/LMMS/lmms/pull/7636) is merged!

I have just merged https://github.com/LMMS/lmms/pull/7636 so this one should be ready to go.

**Comment by bratpeki** [comment]
If noone is working on this, I'll take it up. AFAICT it's just changing all dB-related labels to just "dB" and the rest should display dB instead of linear values in the little infobox.
